processing/stenosis_connector_test.go

```diff
@@ -0,0 +1,220 @@
+package processing
+
+// DCSO FEVER
+// Copyright (c) 2020, DCSO GmbH
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/DCSO/fever/stenosis/task"
+	"github.com/DCSO/fever/types"
+	"github.com/DCSO/fever/util"
+)
+
+// func makeAlertEntry(t *test.T, myTime time.Time, srcIP, dstIP string, srcPort, dstPort int) types.Entry {
```

Comment:
Code can be removed? If not, deserves a comment :)

Reply:
```
// Currently unused. Keeping this here for future randomized tests.
```
added.